File: app/src/main/java/com/argos/android/opencv/interfaces/DialogCallback.java
package com.argos.android.opencv.interfaces;

import android.support.annotation.DrawableRes;

/**
 * Interfacing {@link com.argos.android.opencv.adapter.ImageListAdapter} -> {@link com.argos.android.opencv.activity.MainActivity}
 */

public interface DialogCallback
{
    void launchImageDetection(String feature, @DrawableRes int image);
}

File: app/src/main/java/com/argos/android/opencv/interfaces/DialogCallback.kt
package com.argos.android.opencv.interfaces

import android.support.annotation.DrawableRes

/**
 * Interfacing [com.argos.android.opencv.adapter.ImageListAdapter] -> [com.argos.android.opencv.activity.MainActivity]
 */

interface DialogCallback {
    fun launchImageDetection(feature: String, @DrawableRes image: Int)
}
